include/random_walks/exponential_hmc_leapfrog.hpp

```diff
@@ -0,0 +1,248 @@
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 2021 Vissarion Fisikopoulos
+// Copyright (c) 2021 Apostolos Chalkis
+
+// Licensed under GNU LGPL.3, see LICENCE file
+
+#ifndef RANDOM_WALKS_HAMILTONIAN_HMC_LEAPFROG_HPP
+#define RANDOM_WALKS_HAMILTONIAN_HMC_LEAPFROG_HPP
+
+#include "sampling/sphere.hpp"
+
+
+// Hamiltonian Monte Carlo with leapfrog for sampling from the exponential distribution
+
+struct HMCLeafrogExponential
```

Comment:
Should this code belong to #143 ?

Reply:
This structure removed. The #143 is going to be updated.